src/main/java/dk/cs/aau/huppaal/controllers/HUPPAALController.java

```diff
@@ -239,6 +240,24 @@ public void onChanged(final Change<? extends Component> c) {
 
         initializeReachabilityAnalysisThread();
 
+        File serverFile = UPPAALDriver.getServerFile();
+        StringBuilder sb = new StringBuilder("The file: '" + serverFile.getName() + "' does not exist at location: '" + serverFile.getAbsolutePath() + "'");
+
+        int i = 0;
+        while (i + 49 < sb.length() && (i = sb.lastIndexOf("", i + 49)) != -1) {
+            if(sb.charAt(i+1) == ' '){
+                sb.replace(i, i + 1, "\n");
+            } else {
+                sb.insert( i + 1, "\n");
+            }
+        }
+
+        System.out.println(sb.toString());
```

Comment:
Seems to be debug output?

Reply:
Yes, I forgot to remove the println call, but the above is needed to wrap the text for the tooltip (i couldn't find a better built-in solution)

I have removed the println